src/main/resources/application.yml

```diff
@@ -15,6 +15,9 @@ spring:
     store-type: jdbc
     jdbc:
       initialize-schema: always
+  shell:
```

Comment:
This needs to be disabled by default otherwise the shell starts up on application startup (which we don't want).